packages/lodestar/src/chain/forkChoice/interface.ts

```diff
@@ -2,21 +2,17 @@
  * @module chain/forkChoice
  */
 
-import {Gwei, Slot, ValidatorIndex, Checkpoint} from "@chainsafe/lodestar-types";
+import {Gwei, ValidatorIndex, Checkpoint} from "@chainsafe/lodestar-types";
+import {BlockChainInfo} from "./statefulDag/interface";
```

Comment:
I failed to do that because it's referenced in the outer `interface.ts`. The outer `interface.ts` should be a better place because it's part of the `ILMDGHOST` interface anyway.